setup.py

```diff
@@ -12,7 +12,7 @@
 DESCRIPTION = u'Write Start/Stop events in your SQL logs.'
 PACKAGES = ['django_sql_log']
 REQUIREMENTS = [
-    'django<1.7',
+    'Django',
```

Comment:
good question. the tox.ini configuration sets the django version limit... Is it wrong to allow an untested version here?